src/app/companies/companies-list.ts

```diff
@@ -0,0 +1,18 @@
+import { Company } from './companies.model';
+
+export const COMPANIES_LIST: Company[] = [
+  {
+    name: 'Google',
+    location: 'California',
+    logo:
+      'https://upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Google-favicon-2015.png/150px-Google-favicon-2015.png',
```

Comment:
I think the logos should actually be checked into the repository. Failing that, they should be taken from the official sources, not wikimedia  or some blog.

Reply:
@Airblader - This is a temporary for the PR to get approved, I forgot to mention that. This list should be updated with official list of companies.